ci/pre-prod/tag_image_role/tasks/main.yml

```diff
@@ -14,11 +14,11 @@
     source: local
   with_dict: "{{ image_definitions }}"
 
-- name: Add firecrest-preprod tag to firecrest image
-  docker_image:
-    name: "localhost:5000/{{ item.key }}:{{ commit_id }}"
-    repository: "localhost:5000/{{ item.key }}:firecrest-preprod"
-    force_tag: yes
-    push: yes
-    source: local
-  with_dict: "{{ image_definitions }}"
\ No newline at end of file
+# - name: Add firecrest-preprod tag to firecrest image
```

Comment:
In general, don't fear deleting code.
In the strange case it's actually better to just block comment, leave a comment for posterity saying why so :)